source/server/admin/admin.cc

```diff
@@ -99,10 +99,20 @@ AdminImpl::AdminImpl(const std::string& profile_path, Server::Instance& server,
                       MAKE_ADMIN_HANDLER(server_info_handler_.handlerCerts), false, false),
           makeHandler("/clusters", "upstream cluster status",
                       MAKE_ADMIN_HANDLER(clusters_handler_.handlerClusters), false, false),
-          makeHandler("/config_dump", "dump current Envoy configs (experimental)",
-                      MAKE_ADMIN_HANDLER(config_dump_handler_.handlerConfigDump), false, false),
+          makeHandler(
+              "/config_dump", "dump current Envoy configs (experimental)",
+              MAKE_ADMIN_HANDLER(config_dump_handler_.handlerConfigDump), false, false,
+              {{Admin::ParamDescriptor::Type::String, "resource", "The resource to dump"},
```

Comment:
I realized that and was thinking of doing filling in all the new parameters that were added since I first wrote this code as a follow up. However I'm glad you asked about this because this brings a new complexity. `/logging` is a mutating operation, and thus must be issued as a POST. POST params are transmitted from the browser to the server as POST params in the request body rather than query-params, and there's not currently a path to get get them into the admin handlers. I'm going to have to iterate on this a bit.

I'll get that sorted before asking for the next round of comments.

/wait

Reply:
I decided to back out my addition of POST params in this PR and will follow up. I got this working but it was too hacky. I need to make some infrastructural changes to get POST params to work well, and also to handling the "/logging" semantics for an optional enum param.